### PR TITLE
ENH: Update Readme

### PR DESCRIPTION
**Pull request by syedhamidali**
- [x] Closes #xxxx
- [x] Tests added
- [x] Changes are documented in `history.md`

### CODE REVIEW

**Review by kmuehlbauer — APPROVED** [review]
💯

File: tests/io/test_iris.py [hunk 1]
```diff
@@ -197,3 +198,50 @@ def test_decode_string():
...
+        try:
+            ingest_data_file = IrisIngestDataFile(filename=testfile, loaddata=False)
+            sample_data = np.array([1, 2, 3, 4], dtype=np.uint16)
+            prod_dict = {
+                "func": None,  # Replace with actual function if applicable
+                "dtype": "uint16",
+            }
+            decoded_data = ingest_data_file.decode_data(sample_data, prod_dict)
+            assert isinstance(decoded_data, np.ndarray)
+        except OSError as e:
```
Comment (kmuehlbauer) [inline tests/io/test_iris.py:224]: @syedhamidali Can you shortly explain what the except-branch is doing?

Comment (syedhamidali) [reply tests/io/test_iris.py:224]: Oh, I see. So this iris0_file hasn't been able to load via pytests locally, I put exceptions, but I didn't see it is going to pass the tests anyway. :)

Comment (kmuehlbauer) [reply tests/io/test_iris.py]: Please use `iris0_file` fixture, what did not work locally?

Comment (kmuehlbauer) [reply tests/io/test_iris.py]: The problem might be, that `iris0_file` is no `IrisIngestDataFile` but `IrisRawFile`, no?

**Comment by kmuehlbauer** [comment]
@syedhamidali I'd suggest that you remove all additional changes from this PR and add them as new PR. Then we can merge the Readme changes to have these changes in 0.8.0 (due tomorrow). Thanks!

**Review by kmuehlbauer — CHANGES_REQUESTED** [review]

File: tests/io/test_iris.py (hunk 1 above)
Comment (syedhamidali) [reply tests/io/test_iris.py]: Yes, let me do that instead. I see where the error is. Thanks!

**Comment by kmuehlbauer** [comment]
You can add an entry in the history.md before merge.

**Review by kmuehlbauer — APPROVED** [review]

**Comment by kmuehlbauer** [comment]
Thanks @syedhamidali!

**Pull request merged**